src/components/biggive-timeline-entry/biggive-timeline-entry.tsx

```diff
@@ -0,0 +1,18 @@
+import { Component, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'biggive-timeline-entry',
+  styleUrl: 'biggive-timeline-entry.scss',
+  shadow: true,
+})
+export class BiggiveTimelineEntry {
+  @Prop() entryDate: string;
+
+  @Prop() entryYear: string;
```

Comment:
If `entryDate` and `entryYear` have to be strings can we have a docblock for the expected formats? But I'm confused about why both properties exist, it looks like duplicating information.

Reply:
I have removed the need for entryYear, and added in a docblock